.simplecov

```diff
@@ -0,0 +1,14 @@
+SimpleCov.start 'test_frameworks' do
+  enable_coverage :branch
+  add_group "Aggregations", "lib/arelastic/aggregations"
+  add_group "Arities", "lib/arelastic/arities"
+  add_group "Builders", "lib/arelastic/builders"
+  add_group "Mappings", "lib/arelastic/mappings"
+  add_group "Nodes", "lib/arelastic/nodes"
+  add_group "Queries", "lib/arelastic/queries"
+  add_group "Searches", "lib/arelastic/searches"
+  add_group "Sorts", "lib/arelastic/sorts"
+end
```

Comment:
The groupings seem unnecessary.

Reply:
I found it helpful but it's your call.  I'll remove it if that's what you want.

Reply:
Ok, leave this.